### PR TITLE
Implementing design feedbacks in EIP-1559 V2 flow

### LINKED ISSUE #12771: EIP-1559 v2: small UI tweaks
Small UI tweaks based on @adonesky1 's feedback:

- [x] Remove GWEI amount in Advanced gas fee edit. Just show fiat. Add tooltip that shows full equation. [Reference](https://www.figma.com/file/DFlkrM9JRyP0hDrd4ykMZW?node-id=460:5848#125751387)
- [ ] Add `x` multiplier symbol after multiplier amount. [Reference](https://www.figma.com/file/DFlkrM9JRyP0hDrd4ykMZW?node-id=460:5848#126029907) 
- [ ] Remove multiplier symbol after GWEI amount in `Edit in GWEI` mode. [Reference](https://www.figma.com/file/DFlkrM9JRyP0hDrd4ykMZW?node-id=915:11206#125745003)
- [x] Vertically align gas option emojis with gas option text. [Reference](https://www.figma.com/file/DFlkrM9JRyP0hDrd4ykMZW?node-id=1103:10681#126059483)
- [x] Add hover state to `Edit gas fee` row. Color is #EAF6FF [Reference](https://www.figma.com/file/DFlkrM9JRyP0hDrd4ykMZW/1559-Gas-Controls---v2.1?node-id=460%3A5632)
- [x] Add select state to `Edit gas fee` row. Color is #F2F3F4 [Reference](https://www.figma.com/file/DFlkrM9JRyP0hDrd4ykMZW/1559-Gas-Controls---v2.1?node-id=460%3A5632)
- [x] Left align values to the column headers in `Edit gas fee` screen [Reference](https://www.figma.com/file/DFlkrM9JRyP0hDrd4ykMZW?node-id=1103:10681#126056825) 
- [x] Increase emoji size to 20px in `Edit gas fee` screen. Text labels can remain 12px. [Reference](https://www.figma.com/file/DFlkrM9JRyP0hDrd4ykMZW?node-id=1103:10681#126056639) 
- [x] Make the `Gas option` column slightly smaller and increase the space between the `Time` and `Max fee` columns [Reference](https://www.figma.com/file/DFlkrM9JRyP0hDrd4ykMZW?node-id=1103:10681#126055694)
- [x] Add hover state to each gas option select. Color should be slightly darker PRIMARY3 blue color. [Reference](https://www.figma.com/file/DFlkrM9JRyP0hDrd4ykMZW?node-id=1103:10681#126054900)
- [x]  If there is no “Site” recommended price within the gas options selection, we shouldn’t show that line at all (not grayed out, just not there).
- [x] The “Advanced >” and “Edit” are equally spaced and look like one group. For all options, let’s reduce the space between the label and “>” and also increase the space between “Advanced >” and “Edit”. Mock up below.
Screen Shot 2021-11-30 at 9 36 07 AM

### PR DESCRIPTION
**Pull request by jpuri**
Fixes: #12771
Screenshot 2021-12-02 at 9 43 04 AM
Screenshot 2021-12-02 at 9 37 50 AM

### CODE REVIEW

File: .metamaskrc.dist [hunk 1]
```diff
@@ -2,3 +2,5 @@
 PASSWORD=METAMASK PASSWORD
 INFURA_PROJECT_ID=00000000000
 SEGMENT_WRITE_KEY=
+ONBOARDING_V2=1
```
Comment (mcmire) [inline .metamaskrc.dist:5]: Worth doing this in a different PR so this has more visibility?

File: ui/components/app/edit-gas-fee-popover/edit-gas-item/index.scss [hunk 2]
```diff
@@ -11,12 +11,16 @@
   height: 32px;
   width: 100%;
 
+  &:hover {
```
Comment (mcmire) [inline ui/components/app/edit-gas-fee-popover/edit-gas-item/index.scss:14]: Oh good, I was wondering if this was needed 👍🏻

**Review by mcmire — COMMENTED** [review]
One little thing!

**Review by mcmire — APPROVED** [review]
LGTM!

**Review by adonesky1 — APPROVED** [review]
LGTM!

**Pull request merged**